pkg/validator/labels.go

```diff
@@ -134,7 +134,7 @@ func (h hasAnyOfLabels) Validate(rule rulefmt.Rule) []error {
 			return []error{}
 		}
 	}
-	return []error{fmt.Errorf("missing any of these annotatios `%s`", strings.Join(h.labels, "`,`"))}
+	return []error{fmt.Errorf("missing any of these annotations `%s`", strings.Join(h.labels, "`,`"))}
```

Comment:
```suggestion
	return []error{fmt.Errorf("missing any of these labels `%s`", strings.Join(h.labels, "`,`"))}
```
This was even a typo in a forgotten copypasta :facepalm: